x-pack/plugins/exploratory_view/public/plugin.ts

```diff
@@ -60,7 +60,7 @@ export interface ExploratoryViewPublicPluginsStart {
   discover: DiscoverStart;
   embeddable: EmbeddableStart;
 
-  guidedOnboarding: GuidedOnboardingPluginStart;
+  guidedOnboarding?: GuidedOnboardingPluginStart;
```

Comment:
I know this dependency was there before, but it seems to be unused in this plugin. Probably a copy-paste typo from https://github.com/elastic/kibana/pull/153852

Reply:
Another great catch! You're right, it does not appear to be used, so I removed it.